system_metrics_collector/test/system_metrics_collector/test_linux_memory_measurement.cpp

```diff
@@ -134,8 +133,7 @@ class LinuxMemoryMeasurementTestFixture : public ::testing::Test
 
     ASSERT_FALSE(test_measure_linux_memory_->IsStarted());
 
-    const moving_average_statistics::StatisticData data =
-      test_measure_linux_memory_->GetStatisticsResults();
+    const StatisticData data = test_measure_linux_memory_->GetStatisticsResults();
```

Comment:
```suggestion
    const auto data = test_measure_linux_memory_->GetStatisticsResults();
```

Reply:
Sorry. I missed this, but I didn't want to wait for CI to re-run again. I also think this doesn't reduce that many characters on this line of code, so it would have been an insignificant change.